### PR TITLE
Splitted up PR #15

### PR DESCRIPTION
**Pull request by T-Maxxx**
As you asked.
I did'n removed autoupdate system, I just added "OFFICIAL" symbol so in will be included to updateable binaries and not included to usual builds. Its impossible to debug when it downloads original version, you know :D
Same as PR #15

### CODE REVIEW

File: src/scr_vm_main.c [hunk 1]
```diff
@@ -549,6 +559,8 @@ void Scr_InitFunctions()
     {
         Scr_AddStockFunctions();
         Scr_AddStockMethods();
+        Scr_AddCustomFunctions();
+        Scr_AddStockMethods();
```
Comment (D4edalus) [inline src/scr_vm_main.c]: remove the empty Scr_AddCustomFunctions + Scr_AddStockMethods function please

File: src/scr_vm.h [hunk 2]
```diff
@@ -33,15 +33,30 @@
...
+#define KEY_MASK_RELOAD         16
+#define KEY_MASK_LEANLEFT       64
+#define KEY_MASK_LEANRIGHT      128
+#define KEY_MASK_PRONE          256
+#define KEY_MASK_CROUCH         512
+#define KEY_MASK_JUMP           1024
+#define KEY_MASK_ADS_MODE       2048
+#define KEY_MASK_HOLDBREATH     8192
+#define KEY_MASK_NIGHTVISION    262144
+#define KEY_MASK_ADS            524288
```
Comment (D4edalus) [inline src/scr_vm.h]: move the defines to player.h

File: src/net_reliabletransport.c [hunk 3]
```diff
@@ -273,7 +273,7 @@ void ReliableMessagesTransmitNextFragment(netreliablemsg_t *chan)
 void ReliableMessagesReceiveNextFragment(netreliablemsg_t *chan, msg_t* buf)
 {	
 	int sequence, acknowledge;
-	unsigned int numselectiveack, windowsize, fragmentsize, length, startack;
```
Comment (D4edalus) [inline src/net_reliabletransport.c]: please leave windowsize and fragmentsize in place as it was

**Comment by D4edalus** [comment]
the new Sys_Sleep will be obsolete soon as its gonna be replaced by Sys_SleepSec and Sys_SleepMSec

**Comment by T-Maxxx** [comment]
Done.

**Pull request merged**